---
layout: page
css: "/css/pub.css"
title: AGRO-932 Biometrical genetics and plant breeding
---


- Spring 2020 [Syllabus](img/Syllabus_AGRO932.pdf)
- **Location**: Plant Sciences Hall (room 272)
- **Time**: Tuesday and Thursday 9:00-10:15 am
- **Office Hour**: Thursday 10:15-11:00 am (except the first Thursday of the month) 
- **Prerequisites**: 
  - AGRO 815A, 815B, and 815D, 
  - [AGRO/ANSCI-931](https://jyanglab.com/AGRO-931/), 
  - and	STAT-802 or equivalent, 
  - or permission of the instructor.

------------

## Suggested textbooks and Reading Materials

- Hahn, M. 2019. __Molecular population genetics__. Oxford.
- Bernardo, R. 2020. __Breeding for quantitative traits in plants__. 3rd Ed. Stemma Press. 
- Lynch, M. and B. Walsh. 1998. __Vol. 1: Genetics and analysis of quantitative traits__. Sinauer, MA  

<p align="center">
  <img src="https://i.imgur.com/tSfGg4c.png" height="200px">
</p>

## R and statistics resources

- [Basic R](https://www.statmethods.net/): Quick online reference for data input, basic statistics, and plots.
- [Advanced R](http://adv-r.had.co.nz/) by Hadley Wickham.
- [R Markdown: The definitive Guide](https://bookdown.org/yihui/rmarkdown/) by Yihui Xie et al.,
- [STHDA](http://www.sthda.com/english/): Statistical tools for high-throughput data analysis.

--------------------

# Schedule 2020

### **Week 1**: Introduction and Overview
- 01-14 (Tue): [_Lecture_] Introduction about crop improvement 
  - __[Canvas materials]__ 
- 01-16 (Thu): [_Guest Lecture/Lab_] A basic tutorial on HCC 
  - [[hcc doc](https://hcc.unl.edu/docs/)] and [[hcc-usage in the J. Yang lab](https://jyanglab.com/2018-09-06-hcc/)]

----------------

## Part I. Population Genomics: Crop Domestication and Implications for Plant Breeding

### **Week 2**: Experimental design for selection scan
- 01-21 (Tue): [_Lecture_] Experimental design and basci popgen terminology
  - __[Canvas materials]__ 
- 01-23 (Thu): [_Lab_] Research project management 
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a1.2-lab/proj_managment.html#1)] 

### **Week 3**: Next-generation sequencing and diversity measurement 

- 01-28 (Tue): [_Lecture_] Next-generation sequencing and diversity measurement 
  - __[Hahn 2019, Chapter 4]__
  - __[Canvas materials]__ and [[HTML](https://jyanglab.com/AGRO-932/chapters/a1.1-popgen/hahn_c3_theta.html#1)]
- 01-30 (Thu): [_Lab_] Command line and Slurm job submission
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a1.2-lab/w3lab.html)] and [[ANGSD website](http://www.popgen.dk/angsd/index.php/ANGSD)]

### **Week 4**: Population differentiation
- 02-04 (Tue): [_Lecture_] Measuring population differentiation
  - __[Hahn 2019, Chapter 5]__
  - __[Canvas materials]__ and [[HTML](https://jyanglab.com/AGRO-932/chapters/a1.1-popgen/c5_pops.html#1)]
- 02-06 (Thu): [_Lab_] Calculate Theta and Fst using ANGSD
  - Suggested reading materials: [[Basic Linux/Unix Commands with Examples](https://www.guru99.com/must-know-linux-commands.html)] and [[A Beginner's Guide to Vim](https://www.linux.com/tutorials/vim-101-beginners-guide-vim/)]
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a1.2-lab/w4lab.html#1)]

### **Week 5**: Scan for direct selection
- 02-11 (Tue): [_Lecture_] Direct selection
  - __[Hahn 2019, Chapter 7]__
  - __[Canvas materials]__ and [[HTML](https://jyanglab.com/AGRO-932/chapters/a1.1-popgen/c7_direct_sel.html#1)]
- 02-13 (Thu): [_Lab_] Visualize theta Fst results
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a1.2-lab/w5lab.html#1)]
  - Basic R tutorial: [[Quick-R](https://www.statmethods.net/)] and [[RStudio Cheat Sheets](https://rstudio.com/resources/cheatsheets/)]

### **Week 6**: Linked selection
- 02-18 (Tue): [_Lecture_] Linked selection
  - __[Hahn 2019, Chapter 8]__
  - __[Canvas materials]__ and [[HTML](https://jyanglab.com/AGRO-932/chapters/a1.1-popgen/c8_linked_sel.html#1)]
- 02-20 (Thu): [_Lab_] Breaking down theta statistics into genomic features
  - __[HW1 Due]__
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a1.2-lab/w6lab.html#1)]

----------

## Part II. Conventional Breeding: Mating Design and Heritability Estimation

### **Week 7**: Transition from PopGen to QuantGen
- 02-25 (Tue): [_Lecture_] Definition of breeding populations
  - __[Bernardo, Chapter 2]__
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a2.1-qg/rex1_plant_breeding.html#1)]
- 02-27 (Thu): [_Lab_] Research reporting
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a2.2-lab/w7lab.html)] and [[R Markdown: The Definitive Guide](https://bookdown.org/yihui/rmarkdown/)]

### **Week 8**: Breeding population means and variances
- 03-03 (Tue): [_Lecture_] Phenotypic and genotypic values
  - __[Bernardo, Chapter 3]__ or [[ARGO-931, Ch.7](https://jyanglab.com/AGRO-931/)]
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a2.1-qg/rex2_value.html#1)]
- 03-05 (Thu): [_Lecture_] Phenotypic and genetic variances
  - __[Bernardo, Chapter 6]__ or [[ARGO-931, Ch.8 and Ch.9](https://jyanglab.com/AGRO-931/)]
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a2.1-qg/rex6_variance.html#1)]
  - [__Due date for Mid-term paper__]

### **Week 9**: Mating designs and Estimating Genetic Variances
- 03-10 (Tue): [_Lecture_] Mating design
  - __[Bernardo, Chapter 7]__ or __[Lynch and Walsh, Chapter 18]__
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a2.1-qg/rex7_mating.html#1)]
- 03-12 (Thu): [_Lab_] Genetics variances calculation
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a2.2-lab/w9lab.html)]

### ~~**Week 10**: Genotype x environment interaction~~ Class cancelled due to the Coronavirus
- ~~03-17 (Tue): [_Guest Lecture_]~~
- ~~03-19 (Thu): [_Guest Lab_]~~
  - [__Due date for Mid-term paper review__]

----------

### **Week11**: 
- 3-22 to 3-29: __Spring Break__

---------------------

## Part III. Genome-enabled Breeding and GWAS

- Note: our course switch to Zoom online course because of the Covid-19 pandemic

### **Week 12**: Best linear unbiased prediction
- 03-31 (Tue): [_Lecture_] BLUP introduction and Matrix operations
  - __[Bernardo, Chapter 10]__ or __[Lynch and Walsh, Chapter 8]__
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a2.1-qg/rex10_blup.html)]
- 04-02 (Thu): [_Lecture_] BLUP calculation for breeding populations
  - [[HTML](https://jyanglab.com/AGRO-932/chapters/a2.1-qg/rex10_blup2.html)]

### **Week 13**: Genome-enabled selection
- 04-07 (Tue): [_Lecture_] Genomic selection
  - __[Bernardo, Chapter 11]__ 
  - __[Canvas materials]__ and [[HTML](https://jyanglab.com/AGRO-932/chapters/a2.1-qg/rex11_gs1.html)]
- 04-09 (Thu): [_Lab_] Genomic selection in practice
  - __[Canvas materials]__ and [[HTML](https://jyanglab.com/AGRO-932/chapters/a2.1-qg/rex11_gs2.html)]
  
### **Week 14**: Association mapping via mixed models
- 04-14 (Tue): [_Lecture_] Genome-wide association study
  - __[Bernardo, Chapter 11]__ 
  - [[HTML]()]
- 04-16 (Thu): [_Lab_] 
  - [[HTML]()]

---------------------

## Part IV. Emerging Techonologies in Plant Breeding

### **Week15**: 

- 4-23 (Thu): [__Due date for final paper and final code__]

### **Week16**: 
- 4-28 (Tue): [__Final project presentation__]
- 4-30 (Thu): [__Final project presentation__] [__Due date for final code review__]

------------



<!--

### **Week 2**: Variance components and heritability estimation 
- 10/28 (M): Average effect of an allele, [[HTML]()]
- 10/30 (W): Breeding value, [[HTML]()]
- 11/01 (F): Dominance and Interaction, [[HTML]()]

### **Week 3**: Genetic components of variance
- 11/01 (F): Variance partitioning, [[HTML](chapters/Ch8/Ch8_c1.html#1)], [[HW1](hw/hw2019/AGRO_981_homework3.pdf)]
- 11/04 (M): Variance partitioning (continuous), [[HTML](chapters/Ch8/Ch8_p2.html#1)]
- 11/06 (W): Heritability and Repeatability, [[HTML](chapters/Ch8/Ch8_p3.html#1)]

-------


### **Week 4**: Resemblance between relatives
- 11/08 (F): Covariance, [[HTML](chapters/Ch9/Ch9_c2.html#1)]
- 11/11 (M): Genetic covariance, [[HTML](chapters/Ch9/Ch9_c3.html#1)]
- 11/13 (W): Environmental covariance,  [[HTML](chapters/Ch9/Ch9_c4.html#1)]

### __Exam III__:
- __11/15 (F): 7:30-8:50__

### **Week 5**: Best Linear Unbiased Predictor (BLUP)
- 11/18 (M): Estimation of heritability,  [[HTML](chapters/Ch10/Ch10_c1.html#1)]
- 11/20 (W): Precision of estimates,  [[HTML](chapters/Ch10/Ch10_c2.html#1)], [[lab](chapters/Ch10/lab9-10.html#1)], [peer-review papers posted]

### **Week 6**: Predicting response to selection
- 11/22 (F): The breeder's equation, [[HTML](chapters/Ch11/Ch11_c1.html#1)]
- 11/25 (M): Variability in response, [[HTML](chapters/Ch11/Ch11-c2.html#1)], [__Due date for peer-review__]

 __Thanksgiving Vacation__

### **Week 7**: Selection: Empirical results and interpretation
- 12/02 (M): Asymmetry of responses [[HTML](chapters/Ch12/Ch12-c1.html#1)]
- 12/04 (W): Long-term results [[HTML](chapters/Ch12/Ch12-c2.html#1)], [[HW2](hw/hw2019/AGRO_981_homework4.pdf)]


### **Week 8**: Inbreeding and crossbreeding
- 12/06 (F): Inbreeding depression and heterosis [[HTML](chapters/Ch14/Ch14-c1.html#1)]


### **Week 9**: Correlated traits
- 12/06 (F): Correlated responses to selection [[HTML](chapters/Ch19/Ch19-c1.html#1)]
- 12/09 (M): Index selection [[HTML](chapters/Ch19/Ch19-c2.html#1)]

### **Week 10**: Quantitative trait loci 
- 12/11 (W): QTL: Single-marker analysis [[HTML](chapters/Ch21-2019/Ch21_2019-c1.html#1)]
- 12/13 (M): QTL: Interval Mapping [[HTML](chapters/Ch21-2019/Ch21_2019-c2.html#1)]

### **Week11**: 

 __Thanksgiving Vacation__


### **Week 12**: Correlated traits
- 12/06 (F): Correlated responses to selection [[HTML](chapters/Ch19/Ch19-c1.html#1)]
- 12/09 (M): Index selection [[HTML](chapters/Ch19/Ch19-c2.html#1)]

### **Week 13**: Quantitative trait loci 
- 12/11 (W): QTL: Single-marker analysis [[HTML](chapters/Ch21-2019/Ch21_2019-c1.html#1)]
- 12/13 (M): QTL: Interval Mapping [[HTML](chapters/Ch21-2019/Ch21_2019-c2.html#1)]

### **Week 14**: Inbreeding and crossbreeding
- 12/06 (F): Inbreeding depression and heterosis [[HTML](chapters/Ch14/Ch14-c1.html#1)]


### **Week 15**: Correlated traits
- 12/06 (F): Correlated responses to selection [[HTML](chapters/Ch19/Ch19-c1.html#1)]
- 12/09 (M): Index selection [[HTML](chapters/Ch19/Ch19-c2.html#1)]

### **Week 16**: Quantitative trait loci 
- 12/11 (W): QTL: Single-marker analysis [[HTML](chapters/Ch21-2019/Ch21_2019-c1.html#1)]
- 12/13 (M): QTL: Interval Mapping [[HTML](chapters/Ch21-2019/Ch21_2019-c2.html#1)]

### **Final exam**
- 12/18 (W): 7:30-9:30am
-->
