---
layout: page
title: Q.U.I.T.
subtitle: Paper Mixed up well with Beer (or Non-alcoholic Beverage)
comments: false
---

Inspired by [R.E.H.A.B.](http://www.rilab.org/rehab.html), as well as its fellow journal clubs including [EvolFri](http://evolfri.blogspot.com/), [C.A.M.P.](http://www.public.iastate.edu/~mhufford/HuffordLab/camp/camp.html), [E.H.R.A.B.](http://hagenetics.org/?cat=6), [Detox](http://beissingerlab.org/Detox/), etc., we assemble an informal journal club at UNL to share ideas regarding recent papers or preprints of interest. We focus on, but not limited to, **QU**antitative genetics and genomics, bio-**I**nformatics and breeding **T**echnologies in all aspects.

Anyone is welcome to suggest a paper (by sending an email to _jinliang.yang at unl.edu_) and come to talk with us. Nobody will present or defend the papers. We directly jump right into discussion with the assumption that you have read the paper.
There is no email list; articles are posted below along with the date.

## Notice: Changing time and Location 
We meet every **Friday at 2:00 pm**, at UNL East Campus (Room **PLSH 375**).   

-----------------------------

## 2019

### May

- **05-10-2019**: - **04-26-2019**: Epsilon-genic effects bridge the gap between polygenic and omnigenic complex traits. [C-J Yang et al., 2019](https://www.pnas.org/content/116/12/5643)
> teosinte, domestication, genetics


### April

- **04-26-2019**: Epsilon-genic effects bridge the gap between polygenic and omnigenic complex traits. [Cheng et al., 2019](https://www.biorxiv.org/content/biorxiv/early/2019/04/02/597484.full.pdf)
> polygenic, omnigenic, complex trait

- **04-19-2019**: Deep learning: new computational modelling techniques for genomics. by [Eraslan et al., 2019](https://www.nature.com/articles/s41576-019-0122-6.pdf)
> genomics, review, machine learning

- **04-12-2019**: Fast, scalable prediction of deleterious noncoding variants from functional and population genomic data. by [Huang et al.,](https://www.nature.com/articles/ng.3810)
> evolution, deleterious, genomics

- **04-05-2019**: Promoter-anchored chromatin interactions predicted from genetic analysis of epigenomic data. by [Wu et al.,](https://www.biorxiv.org/content/early/2019/03/18/580993.full.pdf)
> genetics, epigenetics, transcription, HiC

### March

- **03-29-2019**: Widespread signatures of negative selection in the genetic architecture of human complex traits. by [Zeng et al., 2017](https://www.nature.com/articles/s41588-018-0101-4)
> Deleterious alleles, MAF, purifying selection, genomic selection, and polygenicity.

- **03-22-2019**: Dysregulation of expression correlates with rare-allele burden and fitness loss in maize. By [Kremling et al.,](https://www.nature.com/articles/nature25966)
> maize, RNA-seq, deleterious

- **03-15-2019**: In Maize Meeting. No JC.

- **03-08-2019**: Technological advances in maize breeding: past, present and future. By [Andorf et al.,](https://link.springer.com/article/10.1007/s00122-019-03306-3)
> maize, breeding, review



## 2018

### May (Field Season! No Q.U.I.T.!)

- Working on seed preparation, planting and crossing ...


### April

- **04-26-2018**: The genomic impacts of drift and selection for hybrid performance in maize. By [Gerke et al.,](http://www.genetics.org/content/genetics/early/2015/09/18/genetics.115.182410.full.pdf)
> selection, maize, hyrbid


- **04-19-2018**: Identification and validation of genomic regions influencing kernel zinc and iron in maize. By [Hindu et al., 2018](https://www.ncbi.nlm.nih.gov/pubmed/29574570). 
> GWAS, biofortification, mineral

- **04-12-2018**: Field Prep. Skip once.

- **04-05-2018**: A population genetic interpretation of GWAS findings for human quantitative traits. By [Simons et al., 2018](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2002985). And a primer by [Gibson](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2005485) about *Population genetics and GWAS* 
> GWAS, awesome

### March

- **03-29-2018**: The effect of artificial selection on phenotypic plasticity in maize. By [Gage et al., 2017](https://www.nature.com/articles/s41467-017-01450-2) 
> GWAS, GxE, maize

- **03-22-2018**: Dysregulation of expression correlates with rare-allele burden and fitness loss in maize. By [Kremling et al., 2018](https://www.nature.com/articles/nature25966) 
> RNA-seq, fitness, maize

- **03-15-2018**: Microbial genome-enabled insights into plantâ€“microorganism interactions. By [Guttman et al., 2014](https://www.nature.com/articles/nrg3748.pdf) 
> microbiome, review

- **03-08-2018**: Dissecting the genetic architecture controlling highly polygenic traits via simulations in crop species. Contact Jinliang.yang at unl.edu to get the paper.
> GWAS, polygenic

- **03-02-2018**: Quantitative Genetic Analysis of the Maize Leaf Microbiome. by [Wallace et al., 2018](https://www.biorxiv.org/content/early/2018/02/20/268532)
> GWAS, microbiome

### February

- **02-23-2018**: Enhancing resolution of natural methylome reprogramming behavior in plants. by [Sanchez et al., 2017](https://www.biorxiv.org/content/early/2018/02/14/252106)
> methylation

- **02-16-2018**: No Q.U.I.T. Celebrating Lunar New Year! Read below ones:
Genetic architecture: the shape of the genetic contribution to human traits and disease. by [Timpson et al., 2018](https://www.nature.com/articles/nrg.2017.101)
Using partitioned heritability methods to explore genetic architecture. by [Evans and Keller](https://www.nature.com/articles/nrg.2018.6.pdf?WT.ec_id=NRG-201803&spMailingID=55982677&spUserID=MTc2NzI3Njc5NgS2&spJobID=1342043682&spReportId=MTM0MjA0MzY4MgS2)
> GWAS, human

- **02-09-2018**: No Q.U.I.T. because of the weather. Stay warm and read this at home.
A phylogenetic method to perform genome-wide association studies in microbes that accounts for population structure and recombination. by [Collins and Didelot, 2018](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005958)
> GWAS, phylogenetic

- **02-02-2018**: Dynamic DNA methylation reconfiguration during seed development and germination. by [Kawakatsu et al., 2017](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1251-x)
> methylation, arabidopsis


### January

- **01-26-2018**: Genomic features shaping the landscape of meiotic double-strand-break hotspots in maize. by [He et al., 2017](http://www.pnas.org/content/114/46/12231.full.pdf)
> maize, recombination

- **01-19-2018**: Pairwise comparisons across species are problematic when analyzing functional genomic data. by [Dunn et al., 2017](http://www.pnas.org/content/early/2018/01/03/1707515115.long)
> functional genomics, comparative


-----------------------------

## 2017 and before 

[Papers](img/jc_2017) from precious QUIT.


