---
layout: page
title: Publications
css: "/css/pub.css"
subtitle: Plant Quantitative Genetics and Statistical Genomics
---



__[Google Scholar](https://scholar.google.com/citations?hl=en&user=2CiKnzkAAAAJ)__: H-Index [ 21 ], i10-Index [ 28 ], and total citations [ 2023 ] as of May, 2024.

### Preprints

Y. Delen, R. V. Mural, __G. Xu__, __S. Palali Delen__,  J. C. Schnable, __J. Yang__, I. Dweikat, __Dissecting the genetic architecture of sunflower head diameter using genome-wide association study__, _bioRxiv_, 2022.  
[10.1101/2022.10.24.513623](https://www.biorxiv.org/content/10.1101/2022.10.24.513623v1) 






__G. Xu__, __J. Lyu__, T. Obata,  S. Liu, Y. Ge, J. C. Schnable, __J. Yang__, __A historically balanced locus under recent directional selection in responding to changed nitrogen conditions during modern maize breeding__, _bioRxiv_, 2022.  
[10.1101/2022.02.09.479784](https://www.biorxiv.org/content/10.1101/2022.02.09.479784v1) 

C. He, J. D. Washburn, Y. Hao,  Z. Zhang,  __J. Yang__, Sanzhen Liu, __Trait association and prediction through integrative K-mer analysis__, _bioRxiv_, 2021.  
[10.1101/2021.11.17.468725](https://www.biorxiv.org/content/10.1101/2021.11.17.468725v1) 



__J. Li__, D. Cheng, S. Guo, __Z. Yang__, M. Chen, C. Chen, Y. Jiao, W. Li, C. Liu, Y. Zhong, X. Qi, 
__J. Yang__, S. Chen, __Genomic selection to optimize doubled haploid-based hybrid breeding in maize__, _bioRxiv_, 2020.  
[10.1101/2020.09.08.287672](https://www.biorxiv.org/content/10.1101/2020.09.08.287672v1) 

### 2024

__J. Li__, D. Zhang, F. Yang, Q. Zhang, S.i Pan, X. Zhao, Q. Zhang, Y. Han, __J. Yang__, K. Wang, C. Zhao, __[TrG2P: A transfer learning-based tool integrating multi-trait data for accurate prediction of crop yield](https://doi.org/10.1016/j.xplc.2024.100975)__, _Plant Communications_, 2024.  
[Web-based tool]( http://trg2p.ebreed.cn:81) | [GitHub](https://github.com/lijinlong1991/TrG2P)  

Q. Zhang, __Z. Yang__, __J. Yang__, __Dissecting the colocalized GWAS and eQTLs with mediation analysis for high dimensional exposures and confounders__, _Biometrics (accepted)_, 2024.  
[Preprint](https://www.biorxiv.org/content/10.1101/2022.09.08.507169v1) | [GitHub](https://github.com/JIN-HY/SBEyieldprediction)  

__E. Rodene__, G. D. Fernando, V. Piyush, Y. Ge, J. C. Schnable, S. Ghosh, __J. Yang__, __[Image filtering to improve maize tassel detection accuracy using machine learning algorithms](https://www.mdpi.com/1424-8220/24/7/2172)__, _Sensors_, 2024.  
[Preprint](https://www.authorea.com/doi/full/10.22541/au.166758437.77805912)

__H. Jin__, M. C. Tross, R. Tan, L. Newton, R. V. Mural, __J. Yang__, A. M. Thompson, J. C. Schnable, __[Imitating the ‘breeder’s eye’: predicting grain yield from measurements of non-yield traits](https://doi.org/10.1002/ppj2.20102)__, _The Plant Phenome Journal_, 2024.  
[Preprint](https://doi.org/10.1101/2023.11.29.568906) | [GitHub](https://github.com/JIN-HY/SBEyieldprediction)  | [Data](https://doi.org/10.6084/m9.figshare.25432378.v1)

X. He, D. Wang, Y. Jiang, M. Li, M. Delgado-Baquerizo, C. McLaughlin, C. Marcon, M. Baer, L. Guo, Y. A.T. Moya, N. von Wiren, M. Deichmann, G. Schaaf, H.-P. Piepho, __Z. Yang__, __J. Yang__, B. Yim, K. Smalla, S. Goormachtig, F. T. de Vries, H. Huging, R. J. H. Sawers, J. C. Reif, F. Hochholdinger, X. Chen, P. Yu, __[Heritable microbiome variation is correlated with source environment in locally adapted maize varieties](https://www.nature.com/articles/s41477-024-01654-7)__, _Nature Plants_, 2024.  
[Preprint](https://www.biorxiv.org/content/10.1101/2023.01.10.523403v2) | [GitHub](https://github.com/Danning16/MaizeMicrobiome2022)  | [Data: BioProject ID PRJNA889703](https://www.ncbi.nlm.nih.gov/sra)

### 2023

Y. Zou, J. Wang, D. Peng, X. Zhang, L. R. Tembrock, __J. Yang__, J. Zhao, H. Liao, Z. Wu, __[Multi-integrated genomic data for Passiflora foetida provides insights into genome size evolution and floral development in Passiflora](https://link.springer.com/article/10.1186/s43897-023-00076-x)__, _Molecular Horticulture_, 2023.  
[Data](https://ngdc.cncb.ac.cn)


__Z. Yang__, T. Zhao, H. Cheng, and __J. Yang__, __[Microbiome-enabled genomic selection improves prediction accuracy for nitrogen-related traits in maize](https://doi.org/10.1093/g3journal/jkad286)__, _G3_, 2023.  
[Preprint](https://www.biorxiv.org/content/10.1101/2023.03.03.530932v3) | [GitHub](https://github.com/ZhikaiYang/GP_microbiome)  


__S. Palali Delen__, J. Lee, __J. Yang__, __[Improving the metal composition of plants for reduced Cd and increased Zn content: molecular mechanisms and genetic regulations](https://rdcu.be/dp3Gy)__, _Cereal Research Communications_, 2023.  


S. Sun, B. Wang, C. Li, __G. Xu__, __J. Yang__, M. B. Hufford, J. Ross-Ibarra, H. Wang, L. Wang, __[Unraveling prevalence and effects of deleterious mutations in maize elite lines across decades of modern breeding](https://doi.org/10.1093/molbev/msad170)__, _Molecular Biology and Evolution_, 2023.  


T. Hartwig, M. Banf, G. Passaia Prietsch, __J. Yang__, J. Engelhorn, Z. Wang, __[Hybrid allele-specific ChIP-Seq analysis links variation in transcription factor binding to traits in maize](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02909-w)__, _Genome Biology_, 2023.  
[Preprint](https://www.researchsquare.com/article/rs-543958/v1) | [GitHub](https://github.com/mbanf/HaschSeq) | [3 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=2046376427193998427&as_sdt=5)

N. K. Wijewardane, H. Zhang, __J. Yang__, J. C. Schnable, D. P. Schachtman, Y. Ge, __[A leaf-level spectral library to support high throughput plant phenotyping: Predictive accuracy and model transfer](https://doi.org/10.1093/jxb/erad129)__, _Journal of Experimental Botany_, 2023.  
[8 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=11273508399135605886&as_sdt=5)


J. Hao, Y. Yang, S. Futrell, E. A. Kelly, C. M. Lorts, B. Nebie, S. Runo, __J. Yang__, S. Alvarez, J. R. Lasky, D. P. Schachtman, __[CRISPR/Cas9-mediated mutagenesis of _carotenoid cleavage dioxygenase (CCD)_ genes in sorghum alters strigolactone biosynthesis and plant biotic interactions](https://doi.org/10.1094/PBIOMES-08-22-0053-R)__, _Phytobiomes Journal_, 2023.  
[8 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=12968911182331679450&as_sdt=5)


X. Xie, Y. Ge, H. Walia, __J. Yang__, H. Yu, __[Leaf Counting in Monocot Plants using Deep Regression Models](https://doi.org/10.3390/s23041890)__, _Sensors_, 2023.  
[6 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=10999324276339281135&as_sdt=5)

M. W. Grzybowski,  R. V. Mural,  __G. Xu__, J. Turkus,  __J. Yang__, J. C. Schnable, __[A Common Resequencing-Based Genetic Marker Dataset for Global Maize Diversity](https://doi.org/10.1111/tpj.16123)__, _The Plant Journal_, 2023.  
[Preprint](https://www.biorxiv.org/content/10.1101/2022.10.28.514203v1) | [CyVerse](https://datacommons.cyverse.org/browse/iplant/home/shared/Grzybowski_MaizeSNPset_2022) | [Dryad](https://datadryad.org/stash/dataset/doi:10.5061/dryad.bnzs7h4f1) | [14 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=2219072950747194716&as_sdt=5)

__S. Palali Delen__, __G. Xu__, __J. Velazquez-Perfecto__, __J. Yang__, __[Estimating the genetic parameters of yield-related traits under different nitrogen conditions in maize](https://doi.org/10.1093/genetics/iyad012)__, _Genetics_, 2023.  
[Preprint](https://www.biorxiv.org/content/10.1101/2022.08.05.502993v1) | [GitHub](https://github.com/jyanglab/Genetics-parameters-for-N-related-traits) 

D. Li, D. Bai, Y. Tian, Y. Li, C. Zhao, Q. Wang, S. Gou, Y. Gu, X. Luan, R. Wang, __J. Yang__, M. J. Hawkesford, J. C. Schnable, X. Jin, L. Qiu, __[Time series canopy phenotyping enables the identification of genetic variants controlling dynamic phenotypes in soybean](https://doi.org/10.1111/jipb.13380)__, _Journal of Integrative Plant Biology_, 2023.  
[Data](http://onlinelibrary.wiley.com/doi/10.1111/jipb.13380/suppinfo) | [11 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=8054685718312760915&as_sdt=5)



### 2022

X. Liao, Y. Ye, X. Zhang, D. Peng, Me. Hou, G. Fu, J. Tan, J. Zhao, R. Jiang, Y. Xu, J. Liu, __J. Yang__, W. Liu, L. R. Tembrock, G. Zhu, Z. Wu, __[The genomic and bulked segregant analysis of Curcuma alismatifolia revealed its diverse bract pigmentation](https://link.springer.com/article/10.1007/s42994-022-00081-6)__, _aBIOTECH_, 2022.  
[Data](https://ngdc.cncb.ac.cn/gwh) | [11 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=4348408329022405779&as_sdt=5)

L. Yin, __G. Xu__, __J. Yang__, M. Zhao, __[The heterogeneity in the landscape of gene dominance in maize is accompanied by unique chromatin environments](https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msac198/6709529?login=true)__, _Molecular Biology and Evolution_, 2022.  
[Data](https://academic.oup.com/mbe/article/39/10/msac198/6709529#supplementary-data) | [7 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=3441341583601769741&as_sdt=5)


__M. A. Meier__, __G. Xu__, M. G. Lopez-Guerrero, G. Li, C. Smith, B. Sigmon, J. R. Herr, J. R. Alfano, Y. Ge, J. C. Schnable, __J. Yang__, __[Association analyses of host genetics, rootcolonizing microbes, and plant phenotypes under different nitrogen conditions in maize](https://elifesciences.org/articles/75790)__, _eLife_, 2022.  
[Preprint](https://www.biorxiv.org/content/10.1101/2021.11.01.466815v1) | [GitHub](https://github.com/jyanglab/Maize_Rhizobiome_2022) | [NCBI PRJNA772177 (n=3,313 samples)](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA772177) | [25 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=13219677338539149062&as_sdt=5)


S. Deng, __M. A. Meier__, D. Caddell, __J. Yang__, and D. Coleman-Derr, __[Plant Microbiome-Based Genome-Wide Association Studies](https://link.springer.com/protocol/10.1007/978-1-0716-2237-7_20)__, Part of the _Methods in Molecular Biology_ book series, 2022.  


__Z. Yang__, __G. Xu__, Q. Zhang, T. Obata, __J. Yang__, __[Genome-wide mediation analysis: an empirical study to connect phenotype with genotype via intermediate transcriptomic data in maize](https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyac057/6572813?guestAccessKey=62841cd7-b940-470d-b429-bed75e68cf8d)__, _Genetics_, 2022.  
[Preprint](https://www.biorxiv.org/content/10.1101/2021.04.09.439238v1) | [GitHub](https://github.com/ZhikaiYang/mediation) | [GC-MS Data](http://dx.doi.org/10.21228/M86H5T) | [6 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=6551057464498420074&as_sdt=5)


__E. Rodene__, __G. Xu__, __S. P. Delen__, X. Zhao, C. Smith, Y. Ge, J. C. Schnable, __J. Yang__, __[A UAV-based high-throughput phenotyping approach to assess time-series nitrogen responses and identify trait-associated genetic components in maize](https://acsess.onlinelibrary.wiley.com/doi/full/10.1002/ppj2.20030)__, _The Plant Phenome Journal_, 2022.  
[Preprint](https://www.biorxiv.org/content/10.1101/2021.05.24.445447v1) | [GitHub](https://github.com/jyanglab/UAV-based_HTP) | [25 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=16979676899966048479&as_sdt=5)


### 2021

L. F. Samayoa, B. A. Olukolu, C. J. Yang, Q. Chen, M. G. Stetter, A. M. York,, J. de J. Sanchez-Gonzalez, J. C. Glaubitz, P. J. Bradbury, M. C. Romay, Q. Sun, __J. Yang__, J. Ross-Ibarra, E. S. Buckler, J. F. Doebley, and J. B. Holland, __[Domestication Reshaped the Genetic Basis of Inbreeding Depression in a Maize Landrace Compared to its Wild Relative, Teosinte](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009797)__, _PLOS Genetics_, 2021.  
[Preprint](https://www.biorxiv.org/content/10.1101/2021.09.01.458502v1) | [10 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=6632947253215446952&as_sdt=5)


C. Miao, A. Guo, A. Thompson,  __J. Yang__,  Y. Ge,  J. C. Schnable, __[Automation of Leaf Counting in Maize and Sorghum Using Deep Learning](https://acsess.onlinelibrary.wiley.com/doi/10.1002/ppj2.20022)__, _The Plant Phenome Journal_, 2021.  
[Preprint](https://www.biorxiv.org/content/10.1101/2020.12.19.423626v1) | [GitHub](https://github.com/huskermiao/MaizeLeafCounting) | [23 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=16451297146303511073&as_sdt=5)

S. Deng, __G. Xu__, D. Caddell, L. Dahlen, L. Washington, __J. Yang__, D. Coleman-Derr, __[Genome wide association study reveals plant loci controlling heritability of the rhizosphere microbiome](https://www.nature.com/articles/s41396-021-00993-z)__, 
_The ISME Journal_, 2021.  
[Preprint](https://www.biorxiv.org/content/10.1101/2020.02.21.960377v1) | [Github](https://github.com/colemanderr-lab/Deng-2020) | [127 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=9839283634496248436&as_sdt=5)

__M. A. Meier__, M. G. Lopez-Guerrero, M. Guo, M. R. Schmer, J. R. Herr, J. C. Schnable, J. R. Alfano, __J. Yang__, __[Rhizosphere Microbiomes in a Historical Maize/Soybean Rotation System respond to Host Species and Nitrogen Fertilization at Genus and Sub-genus Levels](https://aem.asm.org/content/early/2021/03/27/AEM.03132-20.abstract)__, _Applied and Environmental Microbiology_, 2021.  
[Preprint](https://www.biorxiv.org/content/10.1101/2020.08.10.244384v2) | [GitHub](https://github.com/mandmeier/USDA_CornSoy) | [27 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=8314440354709369054&as_sdt=5)


### 2020

__G. Xu__, __J. Lyu__, Q. Li, H. Liu, D. Wang, M. Zhang, N. M. Springer, J. Ross-Ibarra, __J. Yang__, __[Evolutionary and functional genomics of DNA methylation in maize domestication and improvement](https://www.nature.com/articles/s41467-020-19333-4)__, _Nature Communications_, 2020.  
[Preprint](https://www.biorxiv.org/content/10.1101/2020.03.13.991117v2) | [Github](https://github.com/jyanglab/msfs_teo) | [70 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=987817177097090596&as_sdt=5)

C. Miao, A. Pages, Z. Xu, __E. Rodene__, __J. Yang__, and J. C. Schnable, __[Semantic segmentation of sorghum using hyperspectral data identifies genetic associations](https://spj.sciencemag.org/plantphenomics/2020/4216373/)__, _Plant Phenomics_, 2020.  
[GitHub](https://github.com/huskermiao/Sorghum_Semantic_Segmentation) | [40 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=6760097769110193603&as_sdt=5)


### 2019


C. Miao, T. P. Hoban, A. Pages, Z. Xu, __E. Rodene__, J. Ubbens, I. Stavness, __J. Yang__, and J. C. Schnable, __Simulated plant images improve maize leaf counting accuracy__, _bioRxiv_, 2019.  
[10.1101/706994](https://www.biorxiv.org/content/10.1101/706994v1) 

Y. Ge, A. Atefi, H. Zhang, C. Miao, __R. K. Ramamurthy__, B. Sigmon, __J. Yang__, J. C. Schnable, __[High-throughput analysis of leaf physiological and chemical traits with VIS-NIR-SWIR spectroscopy: A case study with a maize diversity panel](https://plantmethods.biomedcentral.com/articles/10.1186/s13007-019-0450-8)__, _Plant Methods_, 2019.  
[148 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=1137945390019707471&as_sdt=5)


H. Liu, Q. Wang, M. Chen, Y. Ding, X. Yang, J. Liu, X. Li, C. Zhou, Q. Tian, Y. Lu, D. Fan, J. Shi, L. Zhang, C. Kang, M. Sun, F. Li, Y. Wu, Y. Zhang, B. Liu, X.Y. Zhao, Q. Feng, __J. Yang__, B. Han, J. Lai, X.S. Zhang, X. Huang, __[Genome-wide identification and analysis of heterotic loci in three maize hybrids](https://onlinelibrary.wiley.com/doi/abs/10.1111/pbi.13186)__, _Plant Biotechnology Journal_, 2019.  
[44 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=15734942197146441439&as_sdt=5)

C. Miao, __J. Yang__, J. C. Schnable, __[Optimizing the identification of causal variants across varying genetic architectures in crops](https://onlinelibrary.wiley.com/doi/abs/10.1111/pbi.1302)__, _Plant Biotechnology Journal_, 2019.  
[Preprint](https://www.biorxiv.org/content/early/2018/04/29/310391) | [40 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=11827249182260357856&as_sdt=5)


### 2018


R. Shao, H. Zheng, __J. Yang__, S. Jia, T. Liu, Y. Wang, J. Guo, Q. Yang, G. Kang, __[Proteomics Analysis Reveals That Nitric Oxide Regulates Photosynthesis of Maize Seedlings under Water Deficiency](https://www.sciencedirect.com/science/article/pii/S1089860318300958)__, _Nitric Oxide_, 2018.  
[8 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=2779353440279278968&as_sdt=5)

__J. Yang__, C. E. Yeh, __R. K. Ramamurthy__, X. Qi, R. L. Fernando, J. C.M. DekkersD. J. Garrick, D. Nettleton and P. S. Schnable, __[Empirical Comparisons of Different Statistical Models to Identify and Validate Kernel Row Number-Associated Variants from Structured Multiparent Mapping Populations of Maize](http://www.g3journal.org/content/early/2018/09/12/g3.118.200636)__, _G3: Genes Genomes Genetics_, 2018.  
[Github](https://github.com/yangjl/KRN-GWAS) | [Figshare](https://figshare.com/articles/mpirical_Comparisons_of_Different_Statistical_Models_to_Identify_and_Validate_Kernel_Row_Number-Associated_Variants_from_Structured_Multiparent_Mapping_Populations_of_Maize/6902144) | [16 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=996399093583901645&as_sdt=5)


Z. Liang, S. K. Gupta, C. T. Yeh, Y. Zhang, D. W. Ngu, R. Kumar, H. T. Patil, K. D. Mungra, D. V. Yadav, A. Rathore, R. K. Srivastava, R. Gupkta, __J. Yang__, R. K. Varshney, P. S. Schnable, J. C. Schnable, __[Phenotypic data from inbred parents can improve genomic prediction in pearl millet hybrids](http://www.g3journal.org/content/early/2018/05/24/g3.118.200242)__, _G3: Genes Genomes Genetics_, 2018.  
[50 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=6745467722661974167&as_sdt=5)



C. Miao, J. Fang, D. Li, P. Liang, X. Zhang, __J. Yang__, J. C. Schnable, H. Tang, __[Genotype-Corrector: improved genotype calls for genetic mapping in F2 and RIL populations](https://www.nature.com/articles/s41598-018-28294-0)__, _Scientific Reports_, 2018.  
[Github](https://github.com/freemao/Genotype-Corrector) | [25 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=7931817505751558626&as_sdt=5)

M. Bezrutczyk, T. Hartwig, M. Horschman, S. N. Char, __J. Yang__, B. Yang, D. Sosso, W. Frommer, __[Impaired phloem loading in genome-edited triple knock-out mutants of SWEET13 sucrose transporters](https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.15021)__, *New Phytologist*, 2018.  
[Preprint](https://www.biorxiv.org/content/early/2017/10/06/197921) | [135 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=9785166453508429204&as_sdt=5)

P. Bilinski, P. S. Albert, J. J. Berg, J. A. Birchler, M. N. Grote, A. Lorant, J. Quezada, K. Swarts, __J. Yang__, J. Ross-Ibarra, __[Parallel altitudinal clines reveal adaptive evolution of genome size in Zea mays](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007162)__, *PLOS Genetics*, 2018.  
[Preprint](https://www.biorxiv.org/content/early/2017/07/13/134528) | [Github](https://github.com/paulbilinski/GenomeSizeAnalysis) | [122 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=6345641206578854017&as_sdt=5)


### 2017

H.-Y. Lin, Q. Liu, X. Li, __J. Yang__, S. Liu, Y. Huang, M. J. Scanlon, D. Nettleton, P. S. Schnable, __[Substantial contribution of genetic variation in the expression of transcription factors to phenotypic variation revealed by eRD-GWAS](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1328-6)__, *Genome Biology*, 2017.  
[55 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=1407813314092538070&as_sdt=5)

__J. Yang__, S. Mezmouk, A. Baumgarten, E. S. Buckler, K. E. Guill, M. D. McMullen, R. H. Mumm, J. Ross-Ibarra, __[Incomplete dominance of deleterious alleles contribute substantially to trait variation and heterosis in maize](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007019)__, *PLOS Genetics*, 2017.  
[Preprint](https://www.biorxiv.org/content/early/2017/06/09/086132) | [Github](https://github.com/yangjl/GERP-diallel) | [139 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=10986799122751558384&as_sdt=5)

Z. Dong, W. Li, E. Unger-Wallace, __J. Yang__, E. Vollbrecht, G. S. Chuck., __[Ideal crop plant architecture is mediated by tassels replace upper ears1, a BTB/POZ ankyrin repeat gene 5 targeted by TEOSINTE BRANCHED1](http://www.pnas.org/content/early/2017/09/26/1714960114.short)__, *PNAS*, 2017.   
[93 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=9931358203166483013&as_sdt=5)

J. Hao, __J. Yang__, J. Dong, S-Z. Fei, __[Characterization of *BdCBF* genes and Genome-wide Transcriptome Profiling of *BdCBF3*-dependent and -independent Cold Stress Responses in *Brachypodium Distachyon*](http://www.sciencedirect.com/science/article/pii/S0168945217303862?via%3Dihub)__, *Plant Science*, 2017.  
[19 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=3500455729056936894&as_sdt=5)

------------------------

### 2016 and before

__J. Yang__, H. Jiang, C. T. Yeh, J. Yu, J. A. Jeddeloh, D. Nettleton, and P. S. Schnable, __[Extreme-phenotype genome-wide association study (XP-GWAS): A method for identifying trait-associated variants by sequencing pools of individuals selected from a diversity panel](http://doi.wiley.com/10.1111/tpj.13029)__, _The Plant Journal_, 2015.  
[Github](https://github.com/schnablelab/XP-GWAS) | [115 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=4727081806497409997&as_sdt=5)

D. Sosso, D. Luo, Q. Li, J. Sasse, __J. Yang__, G. Gendrot, M. Suzuki, K. E. Koch, D. R. McCarty, P. S. Chourey, P. M. Rogowsky, J. Ross-Ibarra, B. Yang, and W. B. Frommer, __[Seed filling in domesticated maize and rice depends on SWEET-mediated hexose transport](http://www.nature.com/ng/journal/v47/n12/full/ng.3422.html)__, _Nature Genetics_, 2015.  
[405 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=14940779386657346302&as_sdt=5)

S. Leiboff, X. Li, H.-C. Hu, N. Todt, __J. Yang__, X. Li, X. Yu, G. J. Muehlbauer, M. C. P. Timmermans, J. Yu, P. S. Schnable, and M. J. Scanlon, __[Genetic control of morphometric diversity in the maize shoot apical meristem](http://www.nature.com/ncomms/2015/151120/ncomms9974/full/ncomms9974.html)__, _Nature Communications_, 2015.  
[99 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=15673008105270802385&as_sdt=5)


S. Liu, K. Ying, C.-T. Yeh, __J. Yang__, R. Swanson-Wagner, W. Wu, T. Richmond, D. J. Gerhardt, J. Lai, N. Springer, D. Nettleton, J. A. Jeddeloh, and P. S. Schnable, __[Changes in genome content generated via segregation of non-allelic homolog](http://onlinelibrary.wiley.com/doi/10.1111/j.1365-313X.2012.05087.x/abstract)__, _The Plant Journal_, 2012.  
[30 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=12628450597487851072&as_sdt=5)


L. Koesterke, D. Stanzione, M. Vaughn, S. M. Welch, W. Kusnierczyk, __J. Yang__, C.-T. Yeh, D. Nettleton, and P. S. Schnable, __[An Efficient and Scalable Implementation of SNP-Pair Interaction Testing for Genetic Association Studies](http://ieeexplore.ieee.org/document/6008872/?arnumber=6008872)__, *2011 IEEE Int. Symp. Parallel Distrib. Process. Work. Phd Forum, pp. 523–530, May 2011.*  
[9 citations](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=15468679869872245348&as_sdt=5)

------------------------

### Patent Applications

P. S. Schnable, A. OTT, __J. Yang__, __Intercrossed ex-PVP lines__.  *2014. PENDING.*

P. S. Schnable, __J. Yang__, __Identification of QTLs and trait-associated SNPs controlling six yield component traits in maize__. *2013. PENDING.*

P. S. Schnable, __J. Yang__, R. A. Swanson-Wagner, D. Nettleton, __QTL regulating ear productivity traits in maize__. *U.S. Patent No. US8779233B1. granted July 15, 2014.*




