---
layout: page
title: Research Projects
css: "/css/pub.css"
subtitle: Research interests in the J. Yang Lab
---



-------------------

## 1. Improving maize NUE through Omics innovations

Synthetic nitrogen (N) fertilizer is one of the most expensive and energy-intensive
inputs to produce in agricultural production systems worldwide. Inefficient use of N fertilizers in agriculture
leads to a host of environmental and economic concerns. To alleviate the environmental burdens and
increase farming profitability, it is critical to identify sustainable alternatives to improve crop NUE. 
In my group, we aim to improve NUE from the genomics, phenomics, and microbiomics perspectives.

### 1.1 NUE through Genomics Innovation



In the past decades, the genetic gain through continuous
hybrid releases from public and private sectors has contributed substantially to the yield improvement. 
Prior to the 1960s (old-era) in the U.S. Corn Belt, the selection primarily occurred in N-limited
agricultural systems. Since the 1960s or post the Green Revolution (new-era), as inorganic N fertilizers
became increasingly available, maize selection has been mainly conducted in systems where N was
not the limiting constraint on productivity or yield. The shifted N condition has changed the breeders’
preference to select hybrids with high planting density to take advantage of higher rates of N fertilizers,
resulting in a number of physiological and morphological changes in maize. 
However, to what extent the shifted N conditions have reshaped the genomic architecture and regulatory networks in affecting the N assimilation and mobilization has not been systematically studied.
Here, we aim to reveal the genomic changes during recent maize improvement in the oversupplied conditions and validate the functional consequences of these genomic changes.

<p align="center">
  <img height="300" src="https://i.imgur.com/smpnnUV.png">
  <figcaption>Population phenomics and genomics characteristics of maize lines developed from different eras.</figcaption>
</p>

> G. Xu, J. Lyu, T. Obata, S. Liu, Y. Ge, J. C. Schnable, J. Yang, [A historically balanced locus under recent directional selection in responding to changed nitrogen conditions during modern maize breeding](https://www.biorxiv.org/content/10.1101/2022.02.09.479784v1), bioRxiv, 2022.




### 1.2 NUE through Microbiome Innovation



Plant roots are colonized by thousands of microbial species that can improve plant health and performance, such as solubilization of nutrients, competition with pathogens, phytohormone production, and N fixation. Harnessing these beneficial microbes in maize cropping systems provides an attractive opportunity to reduce dependence on chemical N fertilizers.
Our study suggested that the plant host selecting the beneficial
microbes predominantly through negative selection (a form of natural selection to select against
mutations that are deleterious to fitness), resulting in low allele frequencies of the deleterious mutations
across the host genome.
The plant hosts preferentially selected beneficial microbes to provide promising targets for in-depth investigation of the symbiotic relationship, especially the microbes for promoting NUE. 
Here, we will harness the beneficial microbes for maize NUE study from both the plant genetics and plant-associated microbiome perspectives.  

<p align="center">
  <img height="300" src="https://i.imgur.com/3HXTzHC.jpg">
  <figcaption>Phylogenetic tree of taxonomic groups of rhizosphere microbiota.</figcaption>
</p>

> M. A. Meier, G. Xu, M. G. Lopez-Guerrero, G. Li, C. Smith, B. Sigmon, J. R. Herr, J. R. Alfano, Y. Ge, J. C. Schnable, J. Yang, [Maize root-associated microbes likely under adaptive selection by the host to enhance phenotypic performance](https://www.biorxiv.org/content/10.1101/2021.11.01.466815v1), bioRxiv, 2021.

> S. Deng, G. Xu, D. Caddell, L. Dahlen, L. Washington, J. Yang, D. Coleman-Derr, [Genome wide association study reveals plant loci controlling heritability of the rhizosphere microbiome](https://www.nature.com/articles/s41396-021-00993-z), The ISME Journal, 2021.

> M. A. Meier, M. G. Lopez-Guerrero, M. Guo, M. R. Schmer, J. R. Herr, J. C. Schnable, J. R. Alfano, J. Yang, [Rhizosphere Microbiomes in a Historical Maize/Soybean Rotation System respond to Host Species and Nitrogen Fertilization at Genus and Sub-genus Levels](https://journals.asm.org/doi/full/10.1128/AEM.03132-20), Applied and Environmental Microbiology, 2021.


### 1.3 NUE through Phenomics Innovation



In collaboration with several colleagues in phenomics, my group has started to expand the research area on high throughput
phenotyping (HTP), with a focus on in-field phenotyping using
unmanned aerial systems (UAS). Over the years, we have collected UAS imagery data on several
maize diversity panels. To analyze these image data, we developed a UAS image processing pipeline
to extract clipped plot-level images and then apply pixel filtration algorithms to remove the non-foliage
elements.
In the future, we will continue to develop methods to extract physiological and morphological traits (i.e., tassel detection using machine learning methods) from UAS-based imagery data and use these data to map genes and facilitate plant breeding.

<p align="center">
  <img height="300" src="https://i.imgur.com/pkrzKuK.png">
  <figcaption>Field design and data analysis for UAS-based high-throughput phenotyping. </figcaption>
</p>
> E. Rodene, G. Xu, S. P. Delen, X. Zhao, C. Smith, Y. Ge, J. C. Schnable, J. Yang, [A UAV-based high-throughput phenotyping approach to assess time-series nitrogen responses and identify traits-associated genetic components in maize](https://acsess.onlinelibrary.wiley.com/doi/full/10.1002/ppj2.20030), The Plant Phenome Journal, 2022.

-------------------

### 2. Evolutionary and Functional Genomics

Additive and dominant modes of inheritance of deleterious alleles have been investigated in numerous studies; however, very few studies reveal epistasis’s relevance in the evolutionary processes. Biological knowledge and genomics studies suggest that physical interactions,
or functional epistasis, should be prevalent across the genome. For example, the physical interactions
of promoters and enhancers orchestrate spatiotemporal gene expression and transcriptional regulation.
Understanding the role of genomic variation, including the methylation variation, in affecting genome-wide patterns of gene regulation and chromatin interaction is critical for dissecting the phenotypic variation. It is equally important to reveal the selection forces in shaping the patterns of these genomic and epigenomic variations. The knowledge generated by these studies is critical in understanding the evolutionary forces in shaping functional genomics and can be translated into future
crop improvement.
As the genomic profiling of the physical interactions becomes available, our lab will integrate the physical interaction data into the classical epistasis model. We will estimate the effects of the
physical interactions on the phenotypic trait variations and reveal the long-standing questions regarding the relative contributions of epistasis to phenotype.

<p align="center">
  <img height="300" src="https://i.imgur.com/KXgSsTV.png">
  <figcaption>Levels of CG methylation around a differentially methylated region (DMR). </figcaption>
</p>
> G. Xu, J. Lyu, Q. Li, H. Liu, D. Wang, M. Zhang, N. M. Springer, J. Ross-Ibarra, J. Yang, [Evolutionary and functional genomics of DNA methylation in maize domestication and improvement](https://www.nature.com/articles/s41467-020-19333-4), Nature Communications, 2020.

-------------------

### 3. Statistical modeling between genotype and phenotype


The third research area in our lab focuses on bridging the genotype-phenotype (G2P)
divide using statistical modeling approaches. In this direction, we will develop models to integrate various large-scale biological datasets generated in the other two areas, including phenomics, genomics, transcriptomics, methylomics, microbiomics, and functional annotation datasets. 
The overarching goal of this research area is to build tools and statistical
methods to facilitate crop improvement. These tools and methods will help to facilitate plant breeding and
genetics research for a broader community.

<p align="center">
  <img height="300" src="https://i.imgur.com/XeHyQuQ.png">
  <figcaption>A graphical depiction of the high-dimensional genome-wide mediation analysis. </figcaption>
</p>
> Z. Yang, G. Xu, Q. Zhang, T. Obata, J. Yang, [Genome-wide mediation analysis: an empirical study to connect phenotype with genotype via intermediate transcriptomic data in maize](https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyac057/6572813?guestAccessKey=62841cd7-b940-470d-b429-bed75e68cf8d), Genetics, 2022.

