---
layout: page
title: Software
subtitle: Software for Genetics and Genomics
---

Some of the below software packages are intended for internal lab usage. It has not been extensively tested. Use at your own risk.
Most of the software packages are licensed under [GPLv3](https://www.gnu.org/licenses/quick-guide-gplv3.en.html) unless stated otherwise.

### [g3tools](http://jyanglab.com/g3tools/)
- This is an `R packages` with a collection of general purpose tools for **Statistical Genomics**, **Quantitative Genetics**, and **Population Genetics** studies, especially for crop/plant species. 

### [huskeR](http://jyanglab.com/huskeR/)
- This is an `R packages` to generate bioinformatics analysis pipelines and submit jobs to a slurm-based HPC system, especially for [HCC](https://hcc-docs.unl.edu/display/HCCDOC/HCC+Documentation) at UNL.


### [run-latexdiff](http://jyanglab.com/run-latexdiff/)
- A `python` command line tool to create `latexdiff` pdf for different git commits. Modified from [scm-latexdiff](https://bitbucket.org/paulhiemstra/scm-latexdiff/overview). All credits go to scm-latexdiff.

### [pseudoRef](http://jyanglab.com/pseudoRef/)
- It is an `R package` to make a pseudo reference genome subsituted with the SNP variants for given samples.

### [zmSNPtools](https://github.com/yangjl/zmSNPtools)
- A collection of `python`, `perl` and `R` code designed to process a range of basic SNP manipulation tasks.
